lib/shortcode/jetpack-portfolio.php

```diff
@@ -0,0 +1,113 @@
+                    'type'  => 'checkbox',
+                ),
+                array(
+                    'label' => __( 'Display Portfolio Content', 'pbsandwich' ),
+                    'attr'  => 'display_content',
+                    'type'  => 'checkbox',
+                ),
+                array(
+                    'label' => __( 'Entry Types to display', 'pbsandwich' ),
+                    'attr'  => 'include_type',
+                    'type'  => 'text',
+					'description' => __( 'Display Portfolio Types', 'pbsandwich' )'Enter slug names of entries to display. If none entered, defaults to displaying all entries',
+                ),
+                array(
+                    'label' => __( 'Tags to display', 'pbsandwich' ),
+                    'attr'  => 'include_tag',
+                    'type'  => 'text',
+					'description' => __( 'Enter slug names of tags to display. If none entered, defaults to displaying all tags', 'pbsandwich' ),
+                ),
+                array(
+                    'label' => __( 'Columns to display', 'pbsandwich' ),
+                    'attr'  => 'columns',
+                    'type'  => 'select',
+					'description' => __( 'Enter slug names of entries to display. If none entered, defaults to displaying all entries', 'pbsandwich' ),
+					'options' => $numberOfColumns,
+					'default' => '2',
+                ),
+                array(
+                    'label' => __( 'Amount of posts to display', 'pbsandwich' ),
+                    'attr'  => 'showposts',
+                    'type'  => 'text',
+					'description' => __( 'Enter the number of entries to display. If none entered, defaults to 5', 'pbsandwich' ),
+					'value' => '5',
+                ),
+                array(
+                    'label' => __( 'Display Order', 'pbsandwich' ),
+                    'attr'  => 'order',
+                    'type'  => 'select',
+					'description' => __( 'Choose sorting direction. Ascending takes your latest post to the last page, while Descending displays your latest post first.', 'pbsandwich' ),
+					'options' => $displayDir,
+					'default' => 'DESC',
+                ),
+                array(
+                    'label' => __( 'Display Order criteria', 'pbsandwich' ),
+                    'attr'  => 'orderby',
+                    'type'  => 'select',
+					'description' => __( 'Choose the criteria of entries to be sorted by.', 'pbsandwich' ),
+					'options' => $displayOrder,
+					'default' => 'date',
+                ),
+			),
+        )
+    );
+	
+	// Make sure Jetpack is activated
+	if ( ! class_exists( 'Jetpack' ) ) {
+		add_action( 'print_media_templates', 'sandwich_jetpack_disabled' );
+		return;
+	}
+
+	// Make sure the contact form module is turned on
+	if ( ! Jetpack::is_module_active( 'custom-content-types' ) ) {
+		add_action( 'print_media_templates', 'sandwich_jetpack_portfolio_disabled' );
+		return;
+	}
+	
+}
+
+function sandwich_jetpack_disabled() {
```

Comment:
Remove this, just use `sandwich_jetpack_portfolio_disabled`. No need to double this.